### PR TITLE
Console: Setting Default schema to current table's schema for ease of setting Foreign Key

### PR DESCRIPTION
**Pull request by Varun-Choudhary**
Closes: #5376 

### Description
Earlier table's first schema is set as a default schema while setting foreign key. This PR solve that issue by setting default schema to current table's schema for the ease of setting foreign key.

### Affected components

- [X] Console

### CODE REVIEW

**Review by kolharsam — COMMENTED** [review]
These changes look good to me for the `Reference Schema` part alone. One more thing you'll have to ensure is that the `Reference Tables` shown are from the `currentSchema`. And also of course make sure that you can add foreign keys and the whole process is working fine. :)

**Review by kolharsam — COMMENTED** [review]
Apart from one small issue, which I will describe here. I think it is working fine. 👍 
So here's how I spotted the issue:
- From `public` schema, switch to a new schema.
- Click on `Create Table`
- Click to add Foreign Keys
- And you will see that under the `Reference Tables` you'd still be seeing tables from the `public` schema.

File: console/src/components/Services/Data/Add/ForeignKeyWrapper.js [hunk 1]
```diff
@@ -105,6 +106,10 @@ const ForeignKeyWrapper = ({
 
     const expandCallback = () => {
       dispatch(toggleFk(i));
+      const newFks = JSON.parse(JSON.stringify(foreignKeys));
+      newFks[i].refSchemaName = event.target.value;
+      dispatch(setForeignKeys(newFks));
```
Comment (kolharsam) [inline console/src/components/Services/Data/Add/ForeignKeyWrapper.js:111]: I feel we shouldn't mutate `newFks` directly. Here's a suggestion to change that, if required that is:

```suggestion
      const newFks = JSON.parse(JSON.stringify(foreignKeys));
       const updatedFks = {
          ...newFks,
          [i]: {
              ...newFks[i],
              refSchemaName: event.target.value,
          },
      };
      dispatch(setForeignKeys(updatedFks));
```

**Review by kolharsam — COMMENTED** [review]
The code looks good overall. Just the one small suggestion here. 
Still haven't tested it out, will do that and the approve this PR.

**Comment by beerose** [comment]
Closing in favor of https://github.com/hasura/graphql-engine-mono/pull/813

**Pull request closed without merging**